mariza/codepen.html

```diff
@@ -0,0 +1,142 @@
+<header>
+  <div class="header">
+    <center>
+      <h1>Max Gallery</h1>
+    </center>
+    <center> <span class="hours">
+        Open Friday and Saturday <span class="times"> 2:00 p.m. - 5:00 p.m.</span> <span class="appointment">Or by appointment</span> </center>
+    </span>
+    <nav>
+      <ul class="nav-list">
+        <li><a href="#">News</a></li>
+        <li><a href="#">Events</a></li>
+        <li><a href="#">Exhibitions</a></li>
+      </ul>
+    </nav>
+  </div>
+</header>
+<main>
+  <div class="interior">
+    <img class="interior-style" src="https://angeliqueweger.com/temp/gd2-images/gallery3.jpg" alt="Max Gallery Interior shot"></div>
+  <h2>Dance of Conversation</h2>
+  <h3>On display</h3>
+  <h4>Friday March 6, 2020 - Saturday, March 28, 2020</h4>
+  <h3>Opening Reception</h3>
+  <h4>Saturday, March 14, 2020</h4>
+  <div class="press-list">
+    <p>View a press kit on Dropbox, filled with printable images and a press release.</p>
+    <! - - insert button - ->
+      <div class="button"><a href="#">Press Kit</a></div>
```

Comment:
You likely don't need this container `div` here and instead can just add a class directly to the link:
```suggestion
      <a href="#" class="button">Press Kit</a>
```